cloudfoundry/bosh-stemcells-ci
# Migration to use the Community stemcells pipeline and GCP

we migrated the pipeline to the cf foundation hosted community pipeline.
which hosted on gcp.
this pr will
- test bosh on gcp instances
- use gcs hosted storage/buckets for all artifacts and stemcells.
- make building the bosh-agent optional. (as this should be done in a seperate pipeline.
- moving secrets to credhub
- creates stories for the latest usn patches in the comunity github project

things we should take in consideration
- use always the latest bosh agent.
